go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	github.com/giantswarm/microerror v0.3.0
 	github.com/giantswarm/microkit v0.2.2
 	github.com/giantswarm/micrologger v0.5.0
-	github.com/giantswarm/operatorkit/v4 v4.2.0
+	github.com/giantswarm/operatorkit/v4 v4.2.1-0.20210204214713-f1a5d6e536d3
```

Comment:
Will use released version once https://github.com/giantswarm/operatorkit/pull/516 is merged